README.md

```
@@ -1,284 +1,213 @@
 ```
 
+`--optional-flags` takes one JSON object whose keys are the plugin's flags and whose values
+are strings. The flags are listed [below](#plugin-flags); `crane transform optionals
+--plugin-dir ./plugins` prints them with an example each.
+
 ### 3. Review the output
 
-After transform, the output directory contains:
+`crane apply` writes the result under `output/`:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)